src/nodes/chunk_append/chunk_append.c

```diff
@@ -363,6 +363,10 @@ ts_chunk_append_path_create(PlannerInfo *root, RelOptInfo *rel, Hypertable *ht,
 #ifdef USE_ASSERT_CHECKING
 				int nmatches = 0;
 #endif
+				/* Before entering the "DO" loop, check for a valid path entry */
```

Comment:
I am wondering, is the identical check in line 351 still required? Or can it be removed now?

Reply:
@jnidzwetzki ah, yes. Probably not required. Changed.